hail/python/test/hail/fs/test_worker_driver_fs.py

```diff
@@ -135,9 +136,14 @@ def test_can_access_public_blobs():
 
 @run_if_azure
 @fails_local_backend
-def test_qob_can_use_sas_tokens():
+@pytest.mark.asyncio
+async def test_qob_can_use_sas_tokens():
```

Comment:
pytest should be able to pick up on async functions now without the annotation. Is this not true?

Reply:
I didn't think that was true, but apparently you can turn pytest-asyncio into auto mode to let you do that. I don't know how to do that and it doesn't seem to be documented well: https://pytest-asyncio.readthedocs.io/en/latest/concepts.html#auto-mode

Reply:
I believe I've done so in the batch `pytest.ini` file

Reply:
Yeah, you're definitely right. I have deleted every occurrence of pytest.mark.asyncio from the codebase and inserted pytest.ini in `ci/test` and `monitoring/test`.